src/Hosting/Abstractions/src/Microsoft.Extensions.Hosting.Abstractions.csproj

```diff
@@ -8,6 +8,7 @@
     <PackageTags>hosting</PackageTags>
     <RootNamespace>Microsoft.Extensions.Hosting</RootNamespace>
     <DefineConstants Condition="'$(TargetFramework)' == 'netcoreapp3.0'">$(DefineConstants);DISPOSE_ASYNC</DefineConstants>
+    <IsShipping>true</IsShipping>
```

Comment:
Oops. This might need to be backported to release/3.0-preview3 to ensure this package ends up on NuGet.org.

cc @Eilon

Reply:
@muratg / @Tratcher - yes?

Reply:
? This has been shipping since 2.1

Reply:
Yes, if this is currently NOT shipping in preview3, we'll need to backport it.

Reply:
Was this broken just recently and we need to fix? If so yes let's take just that fix in 3.0 preview 3.

Reply:
Looks like this line is not there: https://github.com/aspnet/Extensions/blob/release/3.0-preview3/src/Hosting/Abstractions/src/Microsoft.Extensions.Hosting.Abstractions.csproj


@pakrym are you able to send a PR to preview3 branch as well?

Reply:
Removed here: https://github.com/aspnet/Extensions/commit/a38846cfa13a1e32a5aae90e54dbb71bef20a7db#diff-eadfc5ae0f6d4e8d7fc7de50b06b927cL10

Reply:
Yep, will do.